VERSIONING.md

```diff
@@ -0,0 +1,108 @@
+# Versioning
```

Comment:
As a follow up, it's good to link to this from https://catalog.kpt.dev for discoverability.

Reply:
Actually, the site only shows vX.Y. What's the plan for showing other tags? 

https://screenshot.googleplex.com/amjaGmi5yvJEvCn

Here's is a good place to to have a tool-tip to this doc in context.